iocage/Datasets.py

```diff
@@ -402,6 +402,12 @@ def _get_pool_property(
         prop: str
     ) -> typing.Optional[str]:
 
+        if (pool.status != "ONLINE") and (pool.status != "DEGRADED"):
```

Comment:
This reads nicer as `pool.status not in ('ONLINE', 'DEGRADED')`

Reply:
Will address in a separate PR. Good catch!